test/MintableToken.js

```diff
@@ -23,8 +23,13 @@ contract('Mintable', function(accounts) {
   });
 
   it('should mint a given amount of tokens to a given address', async function() {
-    await token.mint(accounts[0], 100);
-    
+    const result = await token.mint(accounts[0], 100);
+    assert.equal(result.logs[0].event, 'Mint');
```

Comment:
Please use `Array#find` instead of hardcoding the index, like `result.logs.find(e => e.event === 'Mint')`. Same for `Transfer`.

Sorry! Last changes and I'll merge.